src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_correlation_property : SagaPersisterTests<SagaWithoutCorrelationProperty, SagaWithoutCorrelationPropertyData>
```

Comment:
Ok, we can iterate on this as we pull this out as the official test suite for saga persisters. Not blocking for merge.